humanguide/design-concepts/in-app-icons-design.rst

```diff
@@ -0,0 +1,72 @@
+In-app icons design
+===================
+
+Interface Icons
+---------------
+
+Interface icons communicate either status or action displays, such as battery level or an alarm, whilst also enabling the user to perform actions like opening a menu, sharing content or print a document. The interface icons are located in the top bar or in-app icons, such as in the Header or the menus.
+
+|stock_message.svg| |share.svg| |stock_alarm-clock.svg|
+|security-alert.svg| |settings.svg|
+
+See a `list of the icons <http://docs.ubports.com/projects/icons/>`__. available in the system.
+
+Color
+~~~~~
+
+Interface icons can be colourized in qml therefore are designed in #808080 color.
+
+Font
+~~~~
+
+To provide a consistent look to the UI and to make the Suru icons distinctive, the design is based on the same patterns as the Ubuntu font. The font patterns can applied to the symbol to define its contours.
+
+.. figure:: /_static/images/humanguide/14.png
+
+Strokes
+~~~~~~~
+
+Stroke ends are squared but the upper end of vertical strokes is slightly oblique with a +10º angle.
+
+.. figure:: /_static/images/humanguide/15.png
+
+Rotation
+~~~~~~~~
+
+When they are not horizontal or vertical, objects are oriented along a +45º axis.
+
+.. figure:: /_static/images/humanguide/16.png
+
+Corners
+~~~~~~~
+
+Squared corners fit best to depict real objects and curved corners for abstract symbols. The following picture shows a zoom on the default outer and inner curvatures, with nodes and Bezier handle. The default curvature can be scaled down when the shape is smaller.
+
+.. figure:: /_static/images/humanguide/17.png
```

Comment:
This isn't necessarily true for app icons, which are always squircles... Do you mean this only for symbolic icons rather than app icons?

Reply:
It should apply only to objects, not to every line or border. So symbolic icons, the pictogram in app icons, etc...

That's how I got it

Reply:
I overlooked this when compiling the info. I think this will be _deprecated_ soon :)
- Now is (as pointed) inconsistent and not true anymore
- I wanted to move notes app and package icons to use round corner, soo

Sorry for that

Reply:
I've pushed the rest of the changes. I'll look into this during the day